src/I18n/Time.php

```diff
@@ -799,6 +799,18 @@ public static function parseTime($time, $format = null)
     }
 
     /**
+     * Convenience method for getting the remaining time from a given time.
+     *
+     * @param DateTime $datetime The date to get the remaining time from.
+     * @return DateInterval|boolean The DateInterval object representing the difference between the two dates or FALSE on failure.
+     */
+    public static function remainingTime($datetime)
```

Comment:
`fromNow()` might make it clearer what this method does and remove the need to handle the cases @ravage84 and @ionas raised without needing an exception.